### DISCUSSION TITLE
This is the university I go to, but not the campus I'm on

### DISCUSSION THREAD

**Discussion by D2n1e7** [community/community#126068] [GitHub Education]
### Hi everyone,

Hello everyone, I need assistance with an issue I'm having. I've reached out to support multiple times without a response. Could you please advise me on how to resolve this matter?

**Comment by MarkCyber** [comment]
What issue are you having? And what university do you go to? Depending on your needs, there are a few methods you can take. 

First, you can google your specific university campus and see if there is a number associated with it. You can try contacting that number and test the different extensions until you reach someone. Keep note of how you were able to reach them and then use that person to try to get you to the department you need to be in contact with. 

You can also try scheduling something with a counselor and using their connections to help you. A lot of university counselors also offer virtual appointments.

You may also try showing up in person to talk to help desk to figure out how to solve your issue. Some university websites even offer a chatbot directly on their website, too. 

If you need more help, feel free to reply to this comment
